exasol/analytics/schema/values.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+from typing import Any
+
+
+def format_timestamp(timestamp: datetime) -> str:
+    return timestamp.strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
I think, you let the DB create the timestamp with CURRENT_TIMESTAMP

Reply:
This is just for completeness. If you prefer, then I can remove datatime from the list of supported datatypes. Currently, we don't need it.

Reply:
ok, then leave it